src/Umbraco.Web.BackOffice/Controllers/ContentController.cs

```diff
@@ -759,52 +759,50 @@ public IActionResult GetNiceUrl(Udi id)
         return NotFound();
     }
 
+    /// <summary>
+    ///     Gets the children for the content id passed in when sorting
+    /// </summary>
+    /// <returns></returns>
+    [FilterAllowedOutgoingContent(typeof(IEnumerable<ContentItemBasic<ContentPropertyBasic>>), "Items", ActionSort.ActionLetter)]
+    public PagedResult<ContentItemBasic<ContentPropertyBasic>> GetChildrenForSort([FromQuery] GetChildrenFilter request) => GetChildren(request);
+
     /// <summary>
     ///     Gets the children for the content id passed in
     /// </summary>
     /// <returns></returns>
     [FilterAllowedOutgoingContent(typeof(IEnumerable<ContentItemBasic<ContentPropertyBasic>>), "Items")]
-    public PagedResult<ContentItemBasic<ContentPropertyBasic>> GetChildren(
-        int id,
-        string includeProperties,
-        int pageNumber = 0,
-        int pageSize = 0,
-        string orderBy = "SortOrder",
-        Direction orderDirection = Direction.Ascending,
-        bool orderBySystemField = true,
-        string filter = "",
-        string cultureName = "") // TODO: it's not a NAME it's the ISO CODE
```

Comment:
Hey there @AdamPrendergast - nice refactor! However, this is a public method on a public class so this is a breaking change, we can't have that. Can you do it without changing this method signature?

Note: I have not looked at the rest of this PR at all, so I am not sure if it's good or wanted, but I just wanted to flag this problem while skimming.

From what I can see quickly, I think your intentions are good, but this code is consumed by many people, I would suggest sticking with the pattern (as bad as it looks) and adding a new method with a lot of params instead. As I can see you're also adding a new public class, which can then also be adopted by people so we need to be 100% sure we would want to maintain it. (spoiler: I don't think we'll want to or need to).